src/components/ButtonPanel.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import Button from './Button';
+
+function ButtonPanel() {
+  return (
+    <>
+      <div>
+        <Button name="AC" />
+        <Button name="+/-" />
+        <Button name="%" />
+        <Button name="÷" />
+      </div>
+      <div>
+        <Button name="7" />
+        <Button name="8" />
+        <Button name="9" />
+        <Button name="*" />
+      </div>
+      <div>
+        <Button name="4" />
+        <Button name="5" />
+        <Button name="6" />
+        <Button name="-" />
+      </div>
+      <div>
+        <Button name="1" />
+        <Button name="2" />
+        <Button name="3" />
+        <Button name="+" />
+      </div>
+      <div>
+        <Button name="0" />
+        <Button name="." />
+        <Button name="=" />
+      </div>
+    </>
```

Comment:
### [Optional]
Your code appears to be WET-(Write Every Time), Microverse encourage DRY-(Don't Repeat Yourself) code, In your case; you repeated the `Button` component multiple times.

You can use [objects.key](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/keys)  and `map` to loop through the objects. `objects.keys` for the outer loop and `map` for the inner loop.

Example; 
```
const GroupButtons = {
  Group 1: ['AC', '+/-', '%', '÷'],
  Group 2: ['7', '8', '9', 'X'],
   ............
}
```

Reply:
### **Thanks a lot!**

I couldn't think of a way to optimize it, really appreciated :+1: